### PR TITLE
[FIX] subscription documentation

### PR DESCRIPTION
**Pull request by iariasleon**

### CODE REVIEW

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 1]
```diff
@@ -1566,7 +1567,7 @@ It must follow the same representation conventions as context attribute metadata
 A `subject` contains the following subfields:
 
 + `entities`: A list of objects, each one composed of the following subfields (`id`/`idPattern` or type must be present):
-    + `id` or `idPattern`: Id or pattern of the affected entities (optional). Both cannot be used at the same time.
+    + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same time, but at least one of them must be set.
```
Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Lookig for the string (e.g. grep) 

```
`id` or `idPattern`
```
there are 4 occurences in the .apib file. Thus, I understand that this PR should also include the 2 missing ones at this moment.

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 2]
```diff
@@ -1832,18 +1833,12 @@ Response code:
          {
             "type": "Room",
             "id": "DC_S1-D41",
-            "temperature": {
```
Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: This (and the next one) correspond to a very recent change in the .apib. It should desapear after merging develop into hardening/csub_documentation branch.

File: doc/apiary/v2/fiware-ngsiv2-reference.apib (hunk 1 above)
Comment (iariasleon) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Two occurrence in `query` and `discover` operations, but, these cases have other behavior:
>  `id` or `idPattern` (both cannot be used at the same time): specifies the exact ID (in the case of `id`)
     or pattern (in the case of `idPattern`) of the entities to search for. If omitted, it means "any entity
     ID".

Other ocurrense is in `list entities` but in this case both field are optionals

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 3]
```diff
@@ -1833,8 +1834,8 @@ Response code:
             "type": "Room",
             "id": "DC_S1-D41",
             "temperature": {
-              "value": 35.6,
-              "type": "none"
+               "value": 35.6,
```
Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Align (remove one whitespace in this line and the next one)

Comment (iariasleon) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: Fixed in  bc7f6b0

**Comment by fgalan** [comment]
LGTM

**Pull request merged**